packages/libllvm/libcompiler-rt.subpackage.sh

```diff
@@ -25,16 +27,16 @@ termux_step_create_subpkg_debscripts() {
 
 	cat <<- EOF > ./postinst
 	#!$TERMUX_PREFIX/bin/sh
-	find $RT_PATH -type l ! -name "libclang_rt*$TERMUX_ARCH*" -exec rm -rf "{}" \;
-	if [ -e $RT_OPT_DIR ]; then
-	    find $RT_OPT_DIR -type f ! -name "libclang_rt*$TERMUX_ARCH*" -exec ln -sf "{}" $RT_PATH \;
+	if [[ -e $RT_OPT_DIR ]]; then
```

Comment:
That's not gonna work since the `postinst` has a `#!/bin/sh` shebang.
```suggestion
	if [ -e "$RT_OPT_DIR" ]; then
```
The should also quote the variable.